packages/viz/src/lib/errors/styling-error.ts

```diff
@@ -0,0 +1,30 @@
+import CartoError from './carto-error';
+
+/**
+ * Utility to build a CartoError related to Styling errors.
+ *
+ * @return {CartoError} A well formed object representing the error.
+ */
+
+/**
+ * CartoStylingError types:
+ * - [Error]
+ * - [Missing property]
+ *
+ * @name CartoStylingError
+ * @memberof CartoError
+ * @api
+ */
+export class CartoStylingError extends CartoError {
+  constructor(message: string, type = stylingErrorTypes.DEFAULT) {
+    super({ message, type });
+    this.name = 'CartoStylingError';
+  }
+}
+
+export const stylingErrorTypes = {
+  DEFAULT: '[Error]',
```

Comment:
This is the approach we had for carto-vl, and I added a lot of 'types'... do you feel like is it too complex?

Reply:
I think it's good so far. We need to keep it simple, but an enumeration with the current error types is nice.